pkgs/by-name/sh/showtime/package.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitLab,
+  appstream,
+  blueprint-compiler,
+  desktop-file-utils,
+  glib,
+  gobject-introspection,
+  gst_all_1,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "showtime";
+  version = "46.3";
+  pyproject = false;
+
+  src = fetchFromGitLab {
+    owner = "GNOME/Incubator";
```

```suggestion
    group = "GNOME";
    owner = "Incubator";
```

Not sure which is preferred. Papers uses `owner = "GNOME/Incubator";` while decibels uses `group = "GNOME";`.